llvm/test/Transforms/LoopVectorize/AArch64/force-target-instruction-cost.ll

```diff
@@ -151,16 +151,18 @@ exit:
   ret void
 }
 
-define void @test_exit_branch_cost(ptr %dst, i64 %x, i32 %y, ptr %dst.1, i1 %c.4, ptr %src, ptr %dst.3, i1 %c.3, ptr %dst.2) {
+define void @test_exit_branch_cost(ptr %dst, ptr %x.ptr, ptr %y.ptr, ptr %dst.1, i1 %c.4, ptr %src, ptr %dst.3, i1 %c.3, ptr %dst.2) {
```

Comment:
```suggestion
define void @test_exit_branch_cost(ptr %dst, ptr noalias %x.ptr, ptr noalias%y.ptr, ptr %dst.1, i1 %c.4, ptr %src, ptr %dst.3, i1 %c.3, ptr %dst.2) {
```

To avoid new memchecks, (while you are here, could also be done for ptr args, the bounds check aren't important here and just add a bunch of extra IR to check))